src/components/UploadFile/index.tsx

```diff
@@ -70,6 +70,7 @@ const UploadFileComponent: React.FC<Props> = ({
                 {label ? <label>{label}</label> : null}
                 <div className="pg-upload-file__content__form">
                     <input
+                        accept="application/pdf,image/jpeg,image/png"
```

Comment:
Add 'accept' to props (should be optional) and use 'application/pdf,image/jpeg,image/png' as default value

Reply:
Fixed, thanks.